doc/python/hover-text-and-formatting.md

```diff
@@ -83,6 +83,20 @@ fig.update_layout(hovermode="x unified")
 fig.show()
 ```
 
+#### Control hovermode with Dash
+
+[Dash](https://plotly.com/dash/) is the best way to build analytical apps in Python using Plotly figures. To run the app below, run `pip install dash`, click "Download" to get the code and run `python app.py`.
+
+Get started  with [the official Dash docs](https://dash.plotly.com/installation) and **learn how to effortlessly [style](https://plotly.com/dash/design-kit/) & [deploy](https://plotly.com/dash/app-manager/) apps like this with <a style="color:red;" href="https://plotly.com/dash/">Dash Enterprise</a>.**
+
+Change the hovermode below and try hovering over the points:
+
+```python hide_code=true
+from IPython.display import IFrame
+snippet_url = 'https://dash-gallery.plotly.host/python-docs-dash-snippets/'
+IFrame(snippet_url + 'hover-text-and-formatting', width='100%', height=630)
```

Comment:
for this app, please change the options from `x/y/closest` to `x/x unified/closest` ... the y option looks bad and "x unified" looks great :) 

also please change the title of the plot to say "hover over the points to see the change" or something